Library/Homebrew/extend/os/linux/dev-cmd/determine-test-runners.rb

```diff
@@ -0,0 +1,49 @@
+# typed: strict
+# frozen_string_literal: true
+
+class TestRunnerFormula
+  extend T::Sig
+
+  sig { returns(T::Array[TestRunnerFormula]) }
+  def dependents
+    @dependent_hash[ENV["HOMEBREW_SIMULATE_MACOS_ON_LINUX"].present?] ||= with_env(HOMEBREW_STDERR: "1") do
```

Comment:
@Rylan12 how difficult would it be to teach `HOMEBREW_SIMULATE_MACOS_ON_LINUX` to simulate a specific macOS version rather than some unspecified macOS version? That would be pretty useful here.

Reply:
I don't think it would be difficult at all.

Having it simulate the oldest version actually predates the changes I made to `SimulateSystem` but I think it was done intentionally to test formulae on a more extreme OS case...? Not sure if that's right or not.